backend/app/services/auth/verification.py

```
@@ -1,46 +1,75 @@
+    async def get(self, session_id: str) -> Optional[Dict[str, Any]]:
+        """Retrieve session data from Redis."""
+        key = self._get_key(session_id)
+        data_str = await self.redis.get(key)
+
+        if not data_str:
+            return None
+
+        try:
+            return json.loads(data_str)
+        except json.JSONDecodeError:
+            logger.error(f"Failed to decode session data for {session_id}")
```

Ruff (0.14.14)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)